### ISSUE TITLE
warnings when used on Python 3.12

### ISSUE THREAD

**Issue by evgeni** [redhat-performance/tuned#584]
When installing 2.21.0 on my Debian unstable machine, I see the following warnings:

```
/usr/lib/python3/dist-packages/tuned/plugins/plugin_bootloader.py:376: SyntaxWarning: invalid escape sequence '\s'
  self._cmd.add_modify_option_in_file(f, {"set\s+" + consts.GRUB2_TUNED_VAR : "", "set\s+" + consts.GRUB2_TUNED_INITRD_VAR : ""}, add = False)
/usr/lib/python3/dist-packages/tuned/plugins/plugin_bootloader.py:376: SyntaxWarning: invalid escape sequence '\s'
  self._cmd.add_modify_option_in_file(f, {"set\s+" + consts.GRUB2_TUNED_VAR : "", "set\s+" + consts.GRUB2_TUNED_INITRD_VAR : ""}, add = False)
/usr/lib/python3/dist-packages/tuned/plugins/plugin_bootloader.py:406: SyntaxWarning: invalid escape sequence '\s'
  cfg = re.sub(r"^\s*set\s+" + consts.GRUB2_TUNED_VAR + "\s*=.*\n", "", grub2_cfg, flags = re.MULTILINE)
/usr/lib/python3/dist-packages/tuned/plugins/plugin_bootloader.py:408: SyntaxWarning: invalid escape sequence '\s'
  cfg = re.sub(r"^\s*set\s+" + consts.GRUB2_TUNED_INITRD_VAR + "\s*=.*\n", "", grub2_cfg, flags = re.MULTILINE)
/usr/lib/python3/dist-packages/tuned/plugins/plugin_bootloader.py:483: SyntaxWarning: invalid escape sequence '\s'
  (grub2_cfg_new, nsubs) = re.subn(r"\b(set\s+" + opt + "\s*=).*$", r"\1" + "\"" + self._cmd.escape(d[opt]) + "\"", grub2_cfg_new, flags = re.MULTILINE)
/usr/lib/python3/dist-packages/tuned/plugins/plugin_net.py:352: SyntaxWarning: invalid escape sequence '\s'
  "\s+TX:": "\nadaptive-tx:",
/usr/lib/python3/dist-packages/tuned/plugins/plugin_net.py:371: SyntaxWarning: invalid escape sequence '\['
  vl = [v for v in value.split('\n') if len(str(v)) > 0 and not re.search("\[fixed\]$", str(v))]
/usr/lib/python3/dist-packages/tuned/plugins/plugin_scheduler.py:138: SyntaxWarning: invalid escape sequence '\;'
  """
/usr/lib/python3/dist-packages/tuned/profiles/functions/functions.py:49: SyntaxWarning: invalid escape sequence '\:'
  sl = [str(v).replace("\:", ":") for v in sl]
/usr/lib/python3/dist-packages/tuned/utils/commands.py:32: SyntaxWarning: invalid escape sequence '\s'
  return re.sub('\s+', ' ', str(s)).strip()
/usr/lib/python3/dist-packages/tuned/utils/commands.py:438: SyntaxWarning: invalid escape sequence '\,'
  return [str(v).replace("\,", ",") for v in l]
/usr/lib/python3/dist-packages/tuned/utils/config_parser.py:28: SyntaxWarning: invalid escape sequence '\s'
  r'(?P<option>[^' + delims + '\s][^' + delims + ']*)'
/usr/lib/python3/dist-packages/tuned/utils/config_parser.py:29: SyntaxWarning: invalid escape sequence '\s'
  r'\s*(?P<vi>[' + delims + '])\s*'
/usr/lib/python3/dist-packages/tuned/utils/config_parser.py:33: SyntaxWarning: invalid escape sequence '\s'
  r'(?P<option>[^' + delims + '\s][^' + delims + ']*)'
/usr/lib/python3/dist-packages/tuned/utils/config_parser.py:35: SyntaxWarning: invalid escape sequence '\s'
  r'(?P<vi>[' + delims + '])\s*'
/usr/lib/python3/dist-packages/tuned/utils/config_parser.py:40: SyntaxWarning: invalid escape sequence '\s'
  self._re = re.compile("\s+(%s).*" % ")|(".join(list(self._inline_comment_prefixes)))
/usr/lib/python3/dist-packages/tuned/utils/nettool.py:119: SyntaxWarning: invalid escape sequence '\s'
  p_filter = Popen(["sed", "s/^\s*//;s/:\s*/:\\n/g"], \
```

**Comment by zacikpa** [comment]
This was fixed in #570, so I would expect it to appear in the next release.

**Comment by evgeni** [comment]
Ah, perfect! Closing then!
